src/Markdig/Renderers/Html/Inlines/LinkInlineRenderer.cs

```diff
@@ -52,6 +57,11 @@ protected override void Write(HtmlRenderer renderer, LinkInline link)
             }
             else
             {
+                if (AutoRelNoFollow)
```

Comment:
Gotcha, adding now. I might have missed something - but where is the closing `/>` printed for the `renderer.EnableHtmlForInline` case?